app/views/smart_proxies/plugins/_puppet_envs.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if envs.present? %>
+<table class="<%= table_css_classes %> table-fixed">
+  <thead>
+    <th class="col-md-10"><%= _("Environment") %></th>
+    <th class="col-md-2"><%= _("Number of classes") %></th>
+  </thead>
+  <tbody>
+    <% envs.each do |name, count| %>
+      <tr>
+        <td class="ellipsis"><%= name %></td>
+        <td><%= count %></td>
```

Comment:
I still haven't found a good way to display the numbers that clarifies they are the class count. Also, per our discussion, I think we might want to add columns here counting the number of new, updated and obsolete classes, and/or a link to the import page for the specific environment.